indexer-opts/README.md

```diff
@@ -0,0 +1,52 @@
+# indexer-opts - Opts struct for CLI arguments for microindexers
+
+This lib is a shared `clap::Parser` instance that defines CLI arguments for all microindexers.
+
+### Features:
+
+- Basic arguments for indexer to start
+- `init_tracing` for initializing the logs for indexer
+- `StartMode` that allows to choose the strategy for restarts:
+  - `from-latest`
+  - `from-interruption` (default)
+
+### Parameters
+
+- `indexer-id` | **Required** Sets the micro-indexer instance ID (for reading/writing indexer meta-data)
+- `start-block-height` | **Required** Block height to start the stream from (not used if `start_mode == from-latest`)
+- `near-archival-rpc-url` | **Required** NEAR JSON RPC URL
+- `chain-id` | **Required** Chain id: testnet or mainnet, used for NEAR Lake initialization
```

Comment:
There is no way to prevent users from shooting their own legs, unfortunately.

Yeah, we can hardcode RPC URLs to the code itself, but what users should do if they use either RPCaaS or a custom one?

IMO the only way is to follow the "we are all adults here" approach

Reply:
I don't think so. It's the question about interface design. Let's just close the possibility of giving the wrong set of arguments.
The first idea that came to my mind is to have enum
- `--mainnet-rpc-url`
- `--testnet-rpc-url`

And take the chain_id from this enum, together with URL. What do you think?